printing/ppconstr.ml

```diff
@@ -234,6 +234,47 @@ let tag_var = tag Tag.variable
             let f (id,c) = pr_id id ++ str ":=" ++ pr ltop c in
             str"@{" ++ hov 0 (prlist_with_sep pr_semicolon f (List.rev l)) ++ str"}"))
 
+  (* Assuming "{" and "}" brackets, prints
+     - if there is enough room
+       { a; b; c }
+     - otherwise
+       {
+        a;
+        b;
+        c
```

Comment:
Can we get a final `;` too?

Reply:
You mean a final `;` only when `}` is on the next line? That's a very good question. I just look at the most recent of `format.mli` and apparently there is, since 4.08.0, a new `Format.pp_print_custom_break` by @keleshev which would provide this (ocaml/ocaml#2002). So, using 4.08.0, that would be possible.

Reply:
>You mean a final ; only when } is on the next line?

Always is fine too

Reply:
Do you mean that, say, `{ a:=0; }` tends to be now the standard writing anyway?